src/routes/fassungen/[thirties]/FassungskommentarModal.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+	import { Modal } from '@skeletonlabs/skeleton-svelte';
+
+	let { commentary, id, openState = $bindable() } = $props();
+</script>
+
+<Modal
+	defaultOpen={true}
+	onOpenChange={() => (openState = false)}
+	classes="fassungskommentar_modal"
+	triggerBase="btn preset-tonal"
+	contentBase="w-[80vw] card bg-surface-100-900 p-4 space-y-4 shadow-xl max-w-screen-sm"
```

Comment:
Use at least md
```suggestion
	contentBase="w-[80vw] card bg-surface-100-900 p-4 space-y-4 shadow-xl max-w-screen-md"
```
or lg
```suggestion
	contentBase="w-[80vw] card bg-surface-100-900 p-4 space-y-4 shadow-xl max-w-screen-lg"
```

as max width to increase readability